Formula/mas.rb

```diff
@@ -6,8 +6,8 @@ class Mas < Formula
   desc "Mac App Store command-line interface"
   homepage "https://github.com/mas-cli/mas"
   url "https://github.com/mas-cli/mas.git",
-      tag:      "v1.8.7",
-      revision: "4405807010987802c0967bbf349c08808062b824"
+      tag:      "v1.8.8-beta.13",
+      revision: "5a6314fe36056bd077ee0b1c08924b169083b9d1"
```

Comment:
This release isn't going to work. The `root_url` below points to the [`v1.8.7`](https://github.com/mas-cli/mas/releases/tag/v1.8.7) release. This is because the bottles were never built or uploaded anywhere.

Reply:
I created the https://github.com/mas-cli/homebrew-tap/labels/pr-pull label, added it to this PR and the `publish` workflow ran and updated the formula - including the `root_url` field -, created the [mas-1.8.8-beta.13](https://github.com/mas-cli/homebrew-tap/releases/tag/mas-1.8.8-beta.13) release and then closed this PR!

Reply:
The workflow requires the label `pr-pull` to be applied to the bump PR.

That starts a process to update `root_url` & to generate the bottles. I assume the PR might have to be not a draft.

My intention was to possibly automate some or all of the following steps:

- switch PR to not a draft (or maybe submit PR as a non-draft in the first place)
- approve PR and/or apply `pr-pull` label (not sure about the order or if approving must be done in addition to applying the label)

Reply:
@phatblat It worked despite this PR being a draft?

Interesting…

Should I create the PR as a normal one instead of as a draft? I was thinking of doing that, I just wanted us to test how that affected the workflow.